README.md

```diff
@@ -74,22 +73,22 @@ Once complete:
 ```
 cd src/
 ```
-and start Omni Core using ```./mastercored``` (or ```./qt/mastercore-qt``` if built with UI). The inital parse step for a first time run
+And start Omni Core using `./bitcoind` (or `./qt/bitcoin-qt` if built with UI). The inital parse step for a first time run
```

Comment:
We need to find a solution for this (at the bare minimum our release binaries should be omnicored / omnicore-qt)